RecodoDesktop/RecodoDesktop/AuthorizationWindow.xaml.cs

```diff
@@ -13,7 +14,13 @@ public partial class AuthorizationWindow : Window
     {
         public AuthorizationWindow()
         {
-            InitializeComponent();
+            if(!checkSavedToken())
```

Comment:
Let's name all private and public functions from big letter

Reply:
Ok